samples/apps/cap/py/src/DebugLog.py

```diff
@@ -0,0 +1,73 @@
+import threading
+import datetime
+import Config
+from termcolor import colored
+
+# Define log levels as constants
+DEBUG = 0
+INFO = 1
+WARN = 2
+ERROR = 3
+
+# Map log levels to their names
+LEVEL_NAMES = ["DBG", "INF", "WRN", "ERR"]
+LEVEL_COLOR = ["dark_grey", "green", "yellow", "red"]
+
+console_lock = threading.Lock()
+
+
+def Log(level, context, msg):
```

Comment:
Use standard python logging rather than introducing another way to log

Reply:
I tried standard python logging.  It does not support color logging in windows.  It only seems to work in linux/mac.  Some newer logging frameworks might have better support for windows.  Adding an item to todo to follow up after this PR.